### PR TITLE
Update Deployment and Development docs based on new auth changes

### PR DESCRIPTION
**Pull request by PuneetPunamiya**
Signed-off-by: Puneet Punamiya <ppunamiy@redhat.com>

# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/main/standards.md#principles) (if functionality changed/added)
- [ ] Run API Unit Tests, Lint Checks, API Design, Golden Files with `make api-check`
- [ ] Run UI Unit Tests, Lint Checks with `make ui-check`
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/main/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/main/CONTRIBUTING.md) for more details._

### CODE REVIEW

File: docs/DEPLOYMENT.md [hunk 1]
```diff
@@ -211,47 +243,47 @@ kubectl apply -f 02-api/ -n tekton-hub
...
 
-## Deploy UI
+      ```bash
+      kubectl apply -f 04-kubernetes/41-ui-ingress.yaml -n tekton-hub
+      ```
 
 ### Update UI ConfigMap
 
-Edit `config/10-config.yaml` and set your GitHub OAuth Client ID and the API URL.
+Edit `config/10-config.yaml` and by setting the API URL, Oauth URL and Redirect URL.
```
Comment (pratap0007) [inline docs/DEPLOYMENT.md:286]:
```suggestion
Edit `config/03-ui/30-ui--config.yaml` and by setting the API URL, Oauth URL and Redirect URL.
```

File: docs/DEPLOYMENT.md [hunk 2]
```diff
@@ -268,14 +300,14 @@ metadata:
   namespace: tekton-hub
 data:
   API_URL: API URL   <<< Update this by API url
-  GH_CLIENT_ID: GH OAuth Client ID   <<< Update this OAuth client id
   API_VERSION: API VERSION  <<< Update this by API version (For e.g-"v1")
-  GHE_URL: Github Enterprise URL <<< Update this if you are getting logged in via Github Enterprise
+  AUTH_BASE_URL: OAUTH URL << Update this by oauth url
+  REDIRECT_URI: UI URL << Update this ui url
```
Comment (pratap0007) [inline docs/DEPLOYMENT.md:305]:
```suggestion
  REDIRECT_URI: UI URL << Update this by ui url
```

File: docs/DEPLOYMENT.md [hunk 3]
```diff
@@ -211,47 +243,47 @@ kubectl apply -f 02-api/ -n tekton-hub
...
 
-- If deploying on OpenShift:-
+```bash
+curl -k -X GET $(oc get -n tekton-hub routes api --template='https://{{ .spec.host }}/v1/categories')
+```
 
-  ```bash
-  kubectl apply -f 04-openshift/40-api-route.yaml -n tekton-hub
-  ```
+NOTE: At this moment, there are no resources in the db. Only the tags and categories from hub config are added in the db.
```
Comment (piyush-garg) [inline docs/DEPLOYMENT.md:254]: there are no tags in config of Hub

**Pull request merged**